sdk/digitaltwins/Azure.DigitalTwins.Core/perf/Azure.DigitalTwins.Core.Perf/Scenarios/QueryDigitalTwins.cs

```diff
@@ -0,0 +1,66 @@
+﻿//Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Threading;
+using System.Threading.Tasks;
+using Azure.Identity;
+using Azure.Test.Perf;
+
+namespace Azure.DigitalTwins.Core.Perf.Scenarios
+{
+    /// <summary>
+    /// The performance test scenario focused on running queries against digital twins instances.
+    /// </summary>
+    /// <seealso cref="PerfTest{SizeOptions}" />
+    public sealed class QueryDigitalTwins : PerfTest<SizeOptions>
+    {
+        private DigitalTwinsClient _digitalTwinsClient;
+
+        public QueryDigitalTwins(SizeOptions options) : base(options)
+        {
+        }
+
+        public override async Task SetupAsync()
```

Comment:
ideally we would use this method to populate the digital twins instance with resources that we can query.